packages/modules/src/modules/banners/designableBanner/stories/DesignableBanner.stories.tsx

```diff
@@ -5,7 +5,7 @@ import { DefaultTemplate } from './Default';
 import { ConfigurableDesign } from '@sdc/shared/dist/types';
 
 export default {
-    title: 'Banners/Custom',
+    title: 'Banners/Designable',
```

Comment:
now that there are 2 stories, I've moved them to their own section

Reply:
Yep, makes sense to separate to designable.

Reply:
They look great. Is the Guardian svg logo colour always black for these? Just can see that being a constant 'tweak'.

Reply:
Good question. Currently there are three options available for the `guardianRoundel` field:
- default (white G, black background)
- brand (white G, blue background)
- inverse (black G, white backround)

I guess this could evolve...we'll see